lib/commonOpts.ts

```diff
@@ -90,10 +90,18 @@ const getAccountId = (options = {}) => {
   const { portal, account } = options;
 
   if (options.useEnv && process.env.HUBSPOT_PORTAL_ID) {
-    return parseInt(process.env.HUBSPOT_PORTAL_ID, 10);
+    const accountIdFromEnv = parseInt(process.env.HUBSPOT_PORTAL_ID, 10);
+    options.account = accountIdFromEnv;
```

Comment:
just a thought while we're in here making changes, but it might make sense for us to use `accountId` and `portalId` to reduce confusion in our commands. We can keep the user-facing option as `--account`, but my thinking is that `account` is already a somewhat loaded term so being explicit could make things easier on us. "account" could mean the id of the account, the name of the account in the cli config, the name of the account in hubspot, or the metadata for the account in the cli config.

Not a blocker, I just saw changes like [this one](https://github.com/HubSpot/hubspot-cli/pull/1212/files#diff-172a3366dd1c2700e6b704f0589a5a8815fb931c2b9c936da8711642d3076979R86) and my first thought was that I liked it better as `accountId`

Reply:
Resolved in:
https://github.com/HubSpot/hubspot-cli/pull/1212/commits/b831e5b2e302063bd8435cb3f1e39429fb2786e5
https://github.com/HubSpot/hubspot-cli/pull/1212/commits/c00709b062658e6e6331d7b732f6990aefd54396
https://github.com/HubSpot/hubspot-cli/pull/1212/commits/53b8a76d0c06d9cdf2d72165c7399bcf854c8d98
https://github.com/HubSpot/hubspot-cli/pull/1212/commits/e6030caf498302d61e1aa38cbfa5509f499aa185
https://github.com/HubSpot/hubspot-cli/pull/1212/commits/95e168b14d410e56f46f7d98be8cfcbbdfa00890